geoips/commandline/geoips_command.py

```diff
@@ -303,59 +331,192 @@ def _get_registry_by_interface_and_package(self, interface, package_name):
                 interface_registry = None
         return interface_registry
 
-    def _print_plugins_short_format(self, interface, interface_registry):
-        """Print the plugins under a certain interface alongside ancillary information.
+    def _get_headers_by_command(self, args, default_headers):
+        """Retrieve the appropriate headers from a list command with the given args.
 
-        "Short Format" includes these pieces of information:
-            [GeoIPS Package, Interface, Plugin Family, Plugin Name, Relpath]
+        Headers are the top-row of the tabular output from 'list' commands. These are
+        customizable via optional args [--column, --short, --long] and we retrieve
```

Comment:
`--short` doesn't currently appear to be handled.

I think we've said that we need to discuss what is included in the default column set, `--short`, and `--long`. Do we have an issue for that?

Reply:
``--short`` is not handled. There's a topic in #455 which discusses that we'd like to support it, but I didn't add it since we haven't discussed what we'd like included in its output. Personally, I think a lot of the ``list`` commands are already short in their output, and that the ``--columns`` flag is enough to get a filtered output if needed. Happy to implement that in this PR if we'd prefer though. 

All in all, I don't see the use for it, but if we all would prefer to have it, I'm happy to implement it.

Reply:
In that case, can you please remove `--short` form the help messages?

Reply:
Yup! I'll remove that from the help message.

Reply:
Removed!